### ISSUE TITLE
Language support: Amharic

### ISSUE THREAD

**Issue by roskee** [giuspen/cherrytree#2280]
**Is your feature request related to a problem? Please describe.**
I think it is always good to have as many languages as possible. It would be great if `Amharic` language is supported

**Describe the solution you'd like**
I would be very happy to contribute to this issue. But I am not sure of the files that needs to be added. I have checked two locations, [po](https://github.com/giuspen/cherrytree/tree/master/po) and [locale](https://github.com/giuspen/cherrytree/tree/master/pygtk2/locale), but I wanted to be sure.

**Comment by giuspen** [comment]
you are correct the file is generated via command `po/i18n_create_lang_file.py am` but I'm attaching it too to this ticket
am.po is a text file but better edited with the free tool https://poedit.net/
If you reach 100% send the file to me and I'll merge it in
[am.po.zip](https://github.com/giuspen/cherrytree/files/11559290/am.po.zip)
